src/models/organization.ts

```diff
@@ -186,18 +176,13 @@ const getOrganizationModel = function (
 	 * balena.models.organization.remove(123);
 	 */
 	const remove = async function (handleOrId: string | number): Promise<void> {
-		try {
-			const id = await getId(handleOrId);
-			await pine.delete<BalenaSdk.Organization>({
-				resource: 'organization',
-				id,
-			});
-		} catch (err) {
-			if (isNotFoundResponse(err)) {
-				treatAsMissingOrganization(handleOrId, err);
-			}
-			throw err;
-		}
+		const id = (
+			await sdkInstance.models.organization.get(handleOrId, { $select: 'id' })
+		).id;
+		await pine.delete<BalenaSdk.Organization>({
+			resource: 'organization',
+			id,
+		});
```

Comment:
This could be combined into a single API call, but it isn't necessary to do that in a major